bot/resources/tags/return.md

```diff
@@ -1,35 +1,32 @@
 **Return Statement**
 
-When calling a function, you'll often want it to give you a value back. In order to do that, you must `return` it. The reason for this is because functions have their own scope. Any values defined within the function body are inaccessible outside of that function.
-
-*For more information about scope, see `!tags scope`*
+A value created inside a function can't be used outside of it unless you `return` it.
 
 Consider the following function:
```

Comment:
How about we add a "scenario" here, which would be something like:
"Let's suppose we want to calculate the square of a given number and we want to use it later on. It would require having a function that calculates it then `return`s it to us for later usage"

Reply:
It's important that the tag be as brief as possible. What value do you think this additional text contributes when the code examples are already there?

Reply:
For total beginners, code might not be an "obvious" thing. And I believe having context helps them better position themselves & understand it better